plugins/destination/bigquery/client/migrate.go

```diff
@@ -178,10 +178,10 @@ func mergeSchemas(haveSchema, wantSchema bigquery.Schema) (bigquery.Schema, erro
 	for _, f := range haveSchema {
 		if want, ok := wantMap[f.Name]; ok {
 			if want.Type != f.Type {
-				return nil, fmt.Errorf("field %v changed type from %v to %v", f.Name, f.Type, want.Type)
+				return nil, fmt.Errorf("column %v changed type from %v to %v. Try dropping the column and re-running", f.Name, f.Type, want.Type)
 			}
 		} else if f.Required {
```

Comment:
Do we need this `required` check? cc @hermanschaaf

Reply:
Yes, if a required column is not in our schema, all inserts will fail

Reply:
Are we setting it or is this something the user does?

Reply:
It's something a user might do, so we're basically checking that they didn't do that.